railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -57,6 +57,19 @@ def diff(content_path = nil)
         say credentials.content_path.read
       end
 
+      desc "fetch PATH", "Fetch a value in the decrypted credentials"
+      def fetch(path)
+        load_environment_config!
+
+        if (yaml = credentials.read)
+          YAML.load(yaml).dig(*path.split("/")).tap do |value|
+            say value.to_s
+          end
+        else
+          say missing_credentials_message
```

Comment:
```suggestion
          say_error missing_credentials_message
```

Error message should go on `stderr`, not `stdout`, it also should exit with non zero status code, otherwise this is very hard to use in a script.